move-vm-backend/src/lib.rs

```diff
@@ -148,6 +149,30 @@ where
         self.warehouse.apply_changes(changeset)
     }
 
+    /// Publish a package of modules into the storage under the given address.
+    pub fn publish_module_package(
+        &self,
+        package: &[u8],
```

Comment:
I still think if it should be named a package - as well as whether the function should include a package in its name. 
As we've got a bundle (and extension .mvb - move bundle) maybe that should be more accurate. WDYT?

Reply:
I think it would be a good idea!

But we'll handle that change fully separately since it would be a renaming PR (set of PRs :smile:)